### DISCUSSION TITLE
Bash Scripting

### DISCUSSION THREAD

**Discussion by LaskiBear** [Budibase/budibase#3751] [Help]
Can someone give a hint how to get commands working on automation task "bash Scripting" im trying simple curl command and it goes to error and reason is curl command not found.
Looked from discussions that users has used curl in bash script so it should work there ? 
Not that familiar with docker / Linux environments so pretty hopeless with those

**Comment by upgreydd** [comment]
If you use docker version just make Dockerfile and update docker-compose.yml to build image, example Dockerfile:
```
FROM budibase.docker.scarf.sh/budibase/apps

RUN apt update -y && \
    apt install -y curl
```
